pkg/kubectl/cmd/create_deployment.go

```diff
@@ -103,16 +103,16 @@ func generatorFromName(
 
 	switch generatorName {
 	case cmdutil.DeploymentBasicAppsV1Beta1GeneratorName:
-		return &kubectl.DeploymentBasicAppsGeneratorV1{
-			Name:   deploymentName,
-			Images: imageNames,
-		}, true
+		generator := new(kubectl.DeploymentBasicAppsGeneratorV1)
+		generator.Name = deploymentName
+		generator.Images = imageNames
+		return generator, true
```

Comment:
I prefer the original style. Don't use `new` followed by initializing fields with assignments, but use literal instead.